### DISCUSSION TITLE
(rufus) create start menu shortcut

### DISCUSSION THREAD

**Discussion by mica** [chocolatey-community/chocolatey-packages#2487] [Ideas]
### Before starting, ensure that you have done the following.

- [X] I have verified that there is no other issue or discussion covering the feature or enhancement
- [X] I have verified there is no open pull request for the feature or enhancement
- [X] I have verified the package to improve is located in this repository

### What do you want to add?

It would be nice if a start menu shortcut to rufus.exe were created during installation  

### If the feature or enhancement is approved:

- [X] I am willing to create a pull request to implement the feature or enhancement

**Comment by pauby** [comment]
The EXE will be shimmed so can be used from the command line. I don't use Rufus, but if there is a need for it to be in the Start Menu, then it should be added! Will move this to an issue.

**Reply by pauby** [reply]
Having never created an issue from a discussion, I didn't realise it would add me as the person who raised it ...
